core/commonTest/src/contract/list/ImmutableListTest.kt

```diff
@@ -68,6 +68,13 @@ class ImmutableListTest {
         compareLists(list, immList)
     }
 
+    @Test fun emptyListToPersistentList() {
+        val empty = emptyList<Int>()
+        val emptyPersistent = empty.toPersistentList()
+
+        assertTrue(emptyPersistent === empty.toPersistentList())
```

Comment:
There is `assertSame` for this, which should give a _slightly_ better error message should it ever fail.

Reply:
Yeah, my preference is to use `assertSame()` but I noticed that this particular file was using `===` instead of `assertSame()`, and I try not to introduce new things for small changes like this. I'm happy to change it, though.

Reply:
Makes sense. It probably pre-dates instance equality being added to kotlin.test! No need to change, was just making sure it was known.